### PR TITLE
Update to appPattern v2.1.2 - Kmohr-soprasteria/issue127

### PR DESCRIPTION
**Pull request by kmohr-soprasteria**

### CODE REVIEW

File: spec/Accessport2LtpTranslator+forwardings.yaml [hunk 1]
```diff
@@ -29,377 +48,385 @@ forwardings:
...
+          uuid: atlt-1-0-2-op-s-bm-012
         - server-name: /v1/inquire-oam-request-approvals
-          uuid: atlt-1-0-0-op-s-bm-005
+          uuid: atlt-1-0-2-op-s-bm-005
       fc-port-deletion:
       operation-client-deletion:
     initiating-requests:
       # [all oam requests]
     consequent-requests:
       - client-name: AdministratorAdministration://v1/approve-oam-request
```
Comment (PrathibaJee) [inline spec/Accessport2LtpTranslator+forwardings.yaml:155]: client-name shall be AccessAdministration:://v1/approve-basic-auth-request

File: spec/Accessport2LtpTranslator+services.yaml [hunk 2]
```diff
@@ -147,99 +147,99 @@ clients:
...
       remote-protocol: http
       ip-address: 127.0.0.1
-      tcp-port: 3028
-      uuid: atlt-1-0-0-tcp-c-aa-2-0-1-000
+      tcp-port: 3035
+      uuid: atlt-1-0-2-tcp-c-aa-2-1-2-000
     operation-clients:
       own-oam:
         basic:
           - operation-name: /v1/approve-oam-request
```
Comment (PrathibaJee) [inline spec/Accessport2LtpTranslator+services.yaml:198]: operation-name shall be /v1/approve-basic-auth-request

File: spec/Accessport2LtpTranslator+config.json [hunk 3]
```diff
@@ -1367,22 +1417,22 @@
...
         ]
       },
       {
-        "uuid": "atlt-1-0-0-op-c-bs-aa-2-0-1-000",
+        "uuid": "atlt-1-0-2-op-c-bs-aa-2-1-2-000",
         "ltp-direction": "core-model-1-4:TERMINATION_DIRECTION_SINK",
         "client-ltp": [
         ],
         "server-ltp": [
-          "atlt-1-0-0-http-c-aa-2-0-1-000"
```
Comment (PrathibaJee) [inline spec/Accessport2LtpTranslator+config.json:1385]: operation-name shall be /v1/approve-basic-auth-request

**Review by PrathibaJee — CHANGES_REQUESTED** [review]
**Verification checklist :**
**configuration**

- [x] Support for the new APIs
/v1/inquire-basic-auth-approvals			*-op-s-bm-012	ServiceRequestCausesLoggingRequest,ServiceRequestCausesLtpUpdateRequest(I), BasicAuthRequestCausesInquiryForAuthentication(M)
/v1/dispose-remainders-of-deregistered-application	*-op-s-bm-013	ServiceRequestCausesLoggingRequest,ServiceRequestCausesLtpDeletionRequest(I)
/v1/inform-about-preceding-release			*-op-s-bm-014	ServiceRequestCausesLoggingRequest
/v1/update-client-of-subsequent-release			*-op-s-bm-015	ServiceRequestCausesLoggingRequest,ServiceRequestCausesLtpUpdateRequest(I)
- [x] version update in the response profile
- [x] ControlConstruct has no validation error(by checking in the server stub)
- [x] AccessAdministrator new API /v1/approve-basic-auth-request support

**Spec**(Basically comparing basic services of this application with the TAC basic services)
- [x] Register-yourself API + callback change
- [ ] EATL callback changes
- [ ] inquire-basic-auth-approval changes

File: spec/Accessport2LtpTranslator+forwardings.yaml (hunk 1 above)
Comment (kmohr-soprasteria) [reply spec/Accessport2LtpTranslator+forwardings.yaml:155]: fixed

**Pull request merged**